### DISCUSSION TITLE
Is it possible to setup a websockets server with anvil?

### DISCUSSION THREAD

**Discussion by Confucian-e** [foundry-rs/foundry#7522] [Help] [answered]
Is it possible to setup a websockets server with anvil?

**Comment by zerosnacks** [comment, marked as answer]
Yes, the websocket connection is available on `ws://127.0.0.1:8545` when starting Anvil

You can use Alloy's `anvil-node-bindings` if you are using Rust: https://github.com/alloy-rs/alloy/blob/main/crates/node-bindings/src/nodes/anvil.rs
